packages/fortress/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
TIL: if I do <kbd>git mv .eslintrc .eslintrc.json</kbd> but then also modify the .eslintrc.json file, it counts as a file delete+add versus a rename. 🤷

Reply:
What if you edit and then mv?